.graphqlconfig.yml.example

```diff
@@ -0,0 +1,10 @@
+projects:
```

Comment:
it would be nice to add a README.md file to the root of the project explaining a bit more about what's required and how to get it :)

Reply:
done